subsys/partition_manager/Kconfig

```diff
@@ -205,11 +205,13 @@ config PM_OVERRIDE_EXTERNAL_DRIVER_CHECK
 config PM_SRAM_BASE
 	hex
 	default $(dt_node_reg_addr_hex,/soc/memory@21000000) if SOC_NRF5340_CPUNET
+	default 0x20000000 if SOC_NRF54L15_ENGA_CPUFLPR
```

Comment:
I assume we don't have a DT node available for this?

Reply:
It's deleted